seqr/utils/elasticsearch/es_gene_agg_search.py

```diff
@@ -12,9 +14,9 @@ class EsGeneAggSearch(EsSearch):
     CACHED_COUNTS_KEY = None
 
     def aggregate_by_gene(self):
-        searches = {self._search}
+        searches = [self._search]
         for index_searches in self._index_searches.values():
-            searches.update(index_searches)
+            searches = searches + [index_search for index_search in index_searches if index_search not in searches]
```

Comment:
Given that sets don't work for these objects in python 3, why would we assume `in` will work here at all?

Reply:
`set()` requires both `__eq__` and `__hash__` while `in` requires only `_eq_`.

Reply:
in general, its better form to do `searches += ...` than `searches = searches + ...`

Reply:
Okay, I will make the change.